StandAlone/WebcamoidInfo.plist

```diff
@@ -21,7 +21,7 @@
         <key>CFBundleSignature</key>
         <string>????</string>
         <key>LSMinimumSystemVersion</key>
-        <string>10.11</string>
+        <string>10.14</string>
```

Comment:
What about passing ```<string>${MACOSX_DEPLOYMENT_TARGET}</string>``` instead? so that way you won't need to update the the environment variable and he plist file each time.

Reply:
That would work for the CI, but necessitate another change somewhere else to make the [wiki build instructions](https://github.com/webcamoid/webcamoid/wiki/Build-and-install-in-Mac) work properly (since they don't yet define that variable). At the top of my head these would work:

- Add the `export MACOSX_DEPLOYMENT_TARGET="10.14"` line to those instructions too.
- Use the `CMAKE_OSX_DEPLOYMENT_TARGET` CMake variable instead and stick it in the root `CMakeLists.txt`.

The wiki didn't need to define that variable yet, since the CMake variable was then auto-generated and ensure that at least the current OS version can execute the built binary, but with `<string>${MACOSX_DEPLOYMENT_TARGET}</string>` it will be needed during builds in order for the plist to be valid.

Any preferences?

Reply:
Implemented and force-pushed.

Using `<string>${CMAKE_OSX_DEPLOYMENT_TARGET}</string>` (note the "CMAKE") instead ensures that the plist is configured with whichever target CMake actually chose.

In the CI that CMake variable will be determined by the `MACOSX_DEPLOYMENT_TARGET` environment variable.
In local builds that CMake variable will be automatically chosen by CMake, unless the environment variable is set.

Either way the .plist build target should now always match the binary.